src/select-state.js

```diff
@@ -0,0 +1,69 @@
+import { createSelector } from 'reselect';
```

Comment:
Would we want to place these input selectors contextually within their respective selector files as opposed to having them all within a single file?

My other assuming in reviewing these changes is that these changes will supersede the existing selector approach of having a global selector object, correct?